host/src/lib/HelperFunctions.jsx

```diff
@@ -256,6 +284,14 @@ export const determineAnswerType = (answer) => {
   }
 };
 
+/**
+ * This provides the equality check for each normalized answer, taking the type of answer to organize the comparison
+ * for more info see: https://github.com/rightoneducation/righton-app/wiki/Short-Answer-Response-%E2%80%90-Equality-Checks
+ * @param {any} normValue 
+ * @param {any} prevAnswerValue 
```

Comment:
Similar to below, are these really any? Can we have a type or maybe make them generic?